custom/conf/app.ini.sample

```diff
@@ -547,6 +547,9 @@ MAX_FILES = 5
 ; Special supported values are ANSIC, UnixDate, RubyDate, RFC822, RFC822Z, RFC850, RFC1123, RFC1123Z, RFC3339, RFC3339Nano, Kitchen, Stamp, StampMilli, StampMicro and StampNano
 ; For more information about the format see http://golang.org/pkg/time/#pkg-constants
 FORMAT =
+; Location the UI time display i.e. Asia/Shanghai
+; Empty means server's location setting
+UI_LOCATION =
```

Comment:
It would be much better if this would be just `DEFAULT_UI_LOCATION ` and actual value would be taken from per-user setting (like it is with theme). Same for date/time format